synopsys-task/src/synopsys-task/tools-parameter.ts

```diff
@@ -190,7 +193,20 @@ export class SynopsysToolsParameter {
     if (parseToBoolean(inputs.BLACKDUCK_AUTOMATION_PRCOMMENT)) {
       console.info("BlackDuck Automation comment is enabled");
       blackduckData.data.azure = await this.getAzureRepoInfo();
+      const azurePullRequestNumber =
```

Comment:
move this to common method

Reply:
it is part of coverity and blackduck pr comment, if we add it in some method we need to return some value that will cause the issue again. we have to specifically set true only in case of classic editor when there's no pr number from env variable

Reply:
What kind of issue we will get if we move it to a common method?

Reply:
we will surely return some value for **environment** either false or true.
and we have to set it for blackduck/coverity.data.environment. If we set it false default adapter will not be called.